scripts/devenv-builder/configure-composer.sh

```diff
@@ -99,6 +99,38 @@ enable_beta_or_eus_repositories() {
         sudo sed -i "s,dist/rhel${version_id_major}/${version_id}/$(uname -m)/baseos/,${version_id_eus}/rhel${version_id_major}/${version_id}/$(uname -m)/baseos/,g" "${composer_config}"
         sudo sed -i "s,dist/rhel${version_id_major}/${version_id}/$(uname -m)/appstream/,${version_id_eus}/rhel${version_id_major}/${version_id}/$(uname -m)/appstream/,g" "${composer_config}"
     fi
+    # If the host OS is configured to use the internal repo, overwrite the composer_config to match
+    if dnf repolist | grep -q download.eng.brq.redhat.com; then
+        # The gpgkey from /usr/share/osbuild-composer/repositories is valid and common for all repos
+        local -r GPGKEY=$(ARCH=$(uname -m) jq '.[env.ARCH][] | select(.name=="baseos") | .gpgkey' /usr/share/osbuild-composer/repositories/rhel-9.6.json)
```

Comment:
The file version is hard-coded in the json file name. Should we use variables?

Also a nit: GPGKEY is a local variable, so we use lower case.

Reply:
updated.